src/news/2_1_0_release.md

```diff
@@ -0,0 +1,240 @@
+---
+lang: "en-US"
+title: "Diesel 2.1.0"
+css: ../assets/stylesheets/application.css
+include-after: |
+    <script src="../assets/javascripts/application.js"></script>
+---
+
+::: demo
+::: content-wrapper
+::: guide-wrapper
+
+Diesel 2.1.0 contains the contributions of more than 41 people. More than 380 commits were submitted
+over a span of 9 months.
+
+This release contains several new features and improves existing features. It introduces support for generating migrations based on the diff between your `schema.rs` file and your database via Diesel CLI. Diesel derives now provides a `#[derive(MultiConnection)]` derive macro that allows to easily combine different database connections into a single enum, which implements `Connection` on its own. The MySQL backend gets support for `upsert` queries via the `ON DUPLICATE KEYS` syntax. Finally we provide new tooling to improve complex error messages generated for common error cases. Check out our [changelog for a complete list of changes](/changelog.html).
+
+This release wouldn't be possible without the support of our contributors and sponsors. If you want to support diesels development, consider [joining the reviewer team](https://github.com/diesel-rs/diesel/issues?q=reviewer+is%3Aopen), submitting [PR's](https://github.com/diesel-rs/diesel/issues?q=is%3Aissue+is%3Aopen++label%3A%22help+wanted%22+), help writing documentation or [sponsor the maintainers](https://github.com/sponsors/weiznich).
```

Comment:
@Ten0 Do you have a sponsorship link that you want to add here (or at the diesel repo)?

Reply:
No I don't, it's fine, thank you! 😊